app/components/certificate/certificate-display.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Flex,
+  Box,
+  HStack,
+  Text,
+  Heading,
+  Tooltip,
+  IconButton,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionIcon,
+  AccordionPanel,
+  useToast,
+} from '@chakra-ui/react';
+import { DownloadIcon, CopyIcon } from '@chakra-ui/icons';
+import type { Certificate } from '@prisma/client';
+
+interface Props {
+  cert: Certificate;
+  priv: Boolean;
```

Comment:
The naming is not very clear to me, I would suggest to add prefix is and put the whole word, so like `isPrivate`